wire-ios-request-strategy/Sources/Protocols/OTREntity.swift

```diff
@@ -149,13 +149,26 @@ extension OTREntity {
                 in: context
             )
 
-        case .v1, .v2, .v3, .v4, .v5, .v6:
-            guard let payload = Payload.MessageSendingStatus(response) else {
+        case .v1, .v2, .v3:
+            guard let payload = Payload.MessageSendingStatusV1(
+                response,
+                decoder: .defaultDecoder
```

Comment:
Is it intended that a `decoder` is passed in this version but not the `MessageSendingStatusV4` version?

Reply:
no probably just a copy paste mistake, but actually the default used anyway when no decoder is passed.